projects/fal/src/fal/cli/flow_runner.py

```diff
@@ -47,10 +47,8 @@ def run_threaded(
 
 
 def fal_flow_run(parsed: argparse.Namespace) -> int:
-    arg_vars = getattr(parsed, "vars", "{}")
-
     # fal-format Python models
-    generated_models = generate_python_dbt_models(parsed.project_dir, arg_vars)
+    generated_models = generate_python_dbt_models(parsed.project_dir, parsed.vars)
```

Comment:
When are we dropping these?

Reply:
> When are we dropping these?

Did not really underrstand the question.

I just introduced `--vars` support for `fal run` too (wanted to use vars for a different use and added it). But it's really not related to dbt 1.5